src/react-components/src/components/Image/Image.tsx

```diff
@@ -34,5 +38,31 @@ export default function Image(props: ImageProps) {
         alt: isDecorative ? "" : altText,
     };
 
-    return <img {...imageProps} />;
+    return (
+        <>
+            <img {...imageProps} />
```

Comment:
The image would need to be inside the `figure` if there are imageCaption or imageCredit, but no `figure` if there aren't.